Makefile.core.mk

```diff
@@ -1,21 +1,33 @@
-export GO111MODULE ?= on
```

Comment:
no go modules? or is it just in the top level?

Reply:
All container-based builds use go modules, we turn that on container-wide.